src/pages/ApiKeys/KeyItem.tsx

```diff
@@ -24,10 +26,15 @@ const KeyItem = ({
   threads,
 }: Omit<KeyInfo, "key"> & { pubKey: string }) => {
   const [visible, setVisible] = useState(false);
-  const [, actions] = useContext(Context);
-  // TODO: Handle getting current org info
+  const [state, actions] = useContext(Context);
```

Comment:
I don't like having this around, we'll have to find a cleaner way to track your "current" org. We could store this in state. To be honest, we might have to totally do away with path-based params just because of how much more difficult this makes static site deployment.